src/ensembl/src/content/app/browser/drawer/drawer-views/DrawerBookmarks.scss

```diff
@@ -0,0 +1,47 @@
+@import 'src/styles/common';
+
+.drawerTitle {
+  margin: 20px 0 0 30px;
+}
+
+.contentWrapper {
+  height: 80%;
+}
+
+.linksWrapper {
+  margin: 20px 0 0 40px;
+  height: 100%;
+  columns: 4;
+  position: relative;
+  width: calc(100% - 110px);
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
```

Comment:
I tried this but I get something like this:
![Screenshot 2019-10-14 at 15 52 02](https://user-images.githubusercontent.com/3085815/66760748-a01adc80-ee9a-11e9-88da-97456e623129.png)

Reply:
With flex I get this:
![Screenshot 2019-10-31 at 15 54 48](https://user-images.githubusercontent.com/3085815/67963719-f441f280-fbf6-11e9-81d6-587b8e3313c4.png)


But with column fill, I get this:
![Screenshot 2019-10-31 at 15 55 11](https://user-images.githubusercontent.com/3085815/67963735-fc019700-fbf6-11e9-8be5-cfb4caa9eb7f.png)

Reply:
Oh, I see what you mean. Are you saying that if we just leave the columns rule, then they will never grow horizontally to more than 4?

I can't get rid of the feeling that we are using columns and flex incorrectly — it will confuse people who will read this CSS. It confused me 😃  I expected there to be 4 columns, but the top image shows five 😃 

What if we remove the columns altogether then, and only use flex, which, I think was what you initially started with.

Will these styles work: https://jsfiddle.net/4e0k51at/ ?

Reply:
I've removed the columns and set the item width to 25% 👍